src/Microsoft.ML.Data/Microsoft.ML.Data.csproj

```diff
@@ -10,6 +10,7 @@
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonPackageVersion)" />
     <PackageReference Include="System.Collections.Immutable" Version="$(SystemCollectionsImmutableVersion)" />
+    <PackageReference Include="System.Threading.Channels" Version="$(SystemThreadingChannelsPackageVersion)" />
     <PackageReference Include="System.Threading.Tasks.Dataflow" Version="$(SystemThreadingTasksDataflowPackageVersion)" />
```

Comment:
We should be able to remove the dependency on `System.Threading.Tasks.Dataflow` now, right?

(Remove it here and in `build/Dependencies.props`)

Reply:
I think this is still needed. The build yelled when I had it removed 😄

Reply:
What was the error? From what I remember `BufferBlock` was the only thing we were using from `Dataflow`.

Reply:
Actually, I misread the error. It seems that it was still trying to restore the package and since there wasn't a version defined it failed. Latest push should fix that.